scripts/jenkins/ardana/ansible/roles/input_model_generator/templates/mkcloud.config.j2

```diff
@@ -89,7 +92,9 @@ export UPDATEREPOS={{ extra_repos.split(',') | join('+') }}
 export nodenumber={{ controllers | int + computes | int }}
 # export nodenumberlonelynode=
 # export nodenumberironicnode=
-# export clusterconfig=
+{% if controllers | int > 1 %}
+export clusterconfig="services+data+network={{ controllers | int }}"
```

Comment:
I think it doesn't hurt to set this even for non-ha case.

Reply:
Given that it is generated, I'm trying to keep the contents of this file down to a minimum. Besides, a one-node cluster makes no sense.